### ISSUE TITLE
Cannot connect to FTPS

### ISSUE THREAD

**Issue by chinseng85** [cy33hc/vita-ftp-client#3]
Hi, i having problem to connect to my FTPS server. Will you support it in future update?

**Comment by cy33hc** [comment]
I don't know. Vita transfer speed aren't that great already, with SSL enabled it might just make transfers too slow.

**Comment by chinseng85** [comment]
Actually the problem is when I connect to FTPS the Messages displayed "220 You will be disconnected after 60 seconds of inactivity", and there isn't any folder show under the Remote panel

**Comment by chinseng85** [comment]
Another possibility is currently Vita doesn't support SSL socket.

**Comment by cy33hc** [comment]
@chinseng85 Nope. I that's not the problem, in order to support SSL, I will have to code it.

**Comment by chinseng85** [comment]
Are you planning to support SSL in future version?
